AttackItem.h

```diff
@@ -1 +1,9 @@
-#include "Item.h"
\ No newline at end of file
+#pragma once
+
+#include "Item.h"
+
+struct AttackItem : Item
+{
+    AttackItem() : Item("attack item", 2) {}
```

Comment:
This should have a boost of 10, not 2. 

- [ ] adjust boost (2nd param) and revise usage in Utility.cpp